neofetch

```diff
@@ -7858,25 +7858,23 @@ EOF
         ;;
 
         "Haiku"*)
-            set_colors 2 8
+            set_colors 1 3 7 10
             read -rd '' ascii_data <<'EOF'
-${c2}          :dc'
-       'l:;'${c1},${c2}'ck.    .;dc:.
-       co    ${c1}..${c2}k.  .;;   ':o.
-       co    ${c1}..${c2}k. ol      ${c1}.${c2}0.
-       co    ${c1}..${c2}k. oc     ${c1}..${c2}0.
-       co    ${c1}..${c2}k. oc     ${c1}..${c2}0.
-.Ol,.  co ${c1}...''${c2}Oc;kkodxOdddOoc,.
- ';lxxlxOdxkxk0kd${c1}oooll${c2}dl${c1}ccc:${c2}clxd;
-     ..${c1}oOolllllccccccc:::::${c2}od;
-       cx:ooc${c1}:::::::;${c2}cooolcX.
-       cd${c1}.${c2}''cloxdoollc' ${c1}...${c2}0.
-       cd${c1}......${c2}k;${c1}.${c2}xl${c1}....  .${c2}0.
-       .::c${c1};..${c2}cx;${c1}.${c2}xo${c1}..... .${c2}0.
-          '::c'${c1}...${c2}do${c1}..... .${c2}K,
-                  cd,.${c1}....:${c2}O,${c1}
-                    ':clod:'${c1}
-                        ${c1}
+${c3}
+
+
+            MMMM              MMMM
+            MMMM              MMMM
+            MMMM              MMMM
+            MMMM              MMMM
+            MMMM${c4}       .ciO| /YMMMMM*"
+ ${c3}           MMMM${c4}   .cOMMMMM|/MMMMM/`
+  ,         ,iMM|/MMMMMMMMMMMMMMM*
+   `*.__,-cMMMMMMMMMMMMMMMMM/`${c3}.MMM
+            MM${c4}MMMMMMM/`:MMM/  ${c3}MMMM
+            MMMM              MMMM
+            MMMM              MMMM
+            """"              """"
```

Comment:
1. Ascii logos have no padding (whitespace) around any edge.
2. Ascii logos must use colors between 0-7 (though exceptions can be made where realistic).

Reply:
Okay, I have removed the left whitespace padding and changed the color 10 to 2

Is it possible to keep the top padding? The new design has been through a TON of revision already

Reply:
Yes. I am thankful for the effort from everyone involved. :)